pkg/operator/controller/crl/controller.go

```diff
@@ -107,13 +120,17 @@ func (r *reconciler) ingressControllersWithConfigmap(name string) ([]operatorv1.
 	return controllers.Items, nil
 }
 
-// configmapToIngressController maps a configmap to a slice of reconcile requests,
-// one request per ingresscontroller that references the configmap.
-func (r *reconciler) configmapToIngressController(o client.Object) []reconcile.Request {
+// clientCAConfigmapToIngressController maps a CRL configmap to a slice of
+// reconcile requests, one request per ingresscontroller that references the
+// configmap.
+func (r *reconciler) clientCAConfigmapToIngressController(o client.Object) []reconcile.Request {
 	requests := []reconcile.Request{}
-	controllers, err := r.ingressControllersWithConfigmap(o.GetName())
+	if o.GetNamespace() != operatorcontroller.DefaultOperandNamespace {
```

Comment:
openshift's API is still referencing `openshift-config` namespace in the comment ([here](https://github.com/openshift/api/blob/master/operator/v1/types_ingress.go#L647)). I guess it's not a big deal except maybe `oc explain` command which seems to take the descriptions from the comments.

Reply:
The user still creates a configmap in the `openshift-config` namespace.  The clientca-controller copies the configmap from `openshift-config` to the `openshift-ingress` namespace, and then this controller reads it from there.  I've added some additional comments in the latest push; does that clear things up?